core/cap-0050.md

```diff
@@ -0,0 +1,183 @@
+```
+CAP: 0050
+Title: Smart Contract Signatures
+Working Group:
+    Owner: Jonathan Jove <@jonjove>
+    Authors:
+    Consulted:
+Status: Draft
+Created: 2022-05-10
+Discussion: TBD
+Protocol version: TBD
+```
+
+## Simple Summary
+
+This proposal explains how to authorize a smart contract interaction.
+
+## Motivation
+
+Actions in smart contracts must be authorized by an external entity. The
+authorization method can vary by smart contract.
+
+Basic authorization provides the authority to interact with a smart contract in
+the name of an account. For example, in CAP-48 and CAP-49 this level of
```

Comment:
Is this defining "basic authorization"? I can't tell exactly. Maybe a reword is in order. Also "provides the authority" to whom? I guess the obvious answer is "to whomever has basic authorization," but this feels a little cyclical.

Possible reword, just to clarify what I'm getting at,

```suggestion
If an entity has basic authorization, it means it has the authority to interact with a smart contract 
on behalf of an account. For example, in CAP-48 and CAP-49, this level of
```